suzieq/engines/pandas/bgp.py

```diff
@@ -67,9 +78,9 @@ def get(self, **kwargs):
 
         if query_str:
             return mdf.query(query_str).drop(columns=drop_cols,
-                                             errors='ignore')
+                                             errors='ignore')[fields]
         else:
-            return mdf.drop(columns=drop_cols, errors='ignore')
+            return mdf.drop(columns=drop_cols, errors='ignore')[fields]
```

Comment:
A test is failing here because "origPeer" is in `fields` but not it is dropped before the column selection.
```suggestion
            return mdf[fields]
```
Same in the `if` statement 2 lines above.